target/ppc/insn32.decode

```diff
@@ -37,6 +37,9 @@
 &VX             vrt vra vrb
 @VX             ...... vrt:5 vra:5 vrb:5 .......... .   &VX
 
+&VX_uim4        vrt uim vrb
+@VX_uim4        ...... vrt:5 - uim:4 vrb:5 ...........  &VX_uim4
```

Comment:
We should keep just the field definitions here, and put the '-'s (don't care) in the instruction definitions. In this case, they're already there, so just need to replace the single '-' with '.'